src/vs/editor/browser/widget/diffEditorWidget.ts

```diff
@@ -1135,7 +1168,13 @@ export class DiffEditorWidget extends Disposable implements editorBrowser.IDiffE
 		result.revealHorizontalRightPadding = EditorOptions.revealHorizontalRightPadding.defaultValue + DiffEditorWidget.ENTIRE_DIFF_OVERVIEW_WIDTH;
 		result.scrollbar!.verticalHasArrows = false;
 		result.extraEditorClassName = 'modified-in-monaco-diff-editor';
-		return result;
+		return {
```

Comment:
Duplicate of lines 1156 - 1162.  One could add a private helper function e.g.

```
function addDimension(result) : editorBrowser.IEditorConstructionOptions {
return {
			...result,
			dimension: {
				height: 0,
				width: 0
			}
		};
```

Then call the helper from both places e.g.

```
return addDimension(result);
```

```